src/components/search/SimpleSearchRow.tsx

```diff
@@ -0,0 +1,97 @@
+import { Grid } from '@mui/material'
+import { useState } from 'react'
+import { DisplayField } from '../../types/Fields'
+import { SearchParameter } from '../../types/SimpleSearch'
+import SearchDateInput from './SearchDateInput'
+import SearchDropdown from './SearchSelectField'
+import SearchTextInput from './SearchTextInput'
+
+interface SimpleSearchRowProps {
+  field: DisplayField
+  parameter: SearchParameter
+  index: number
+  onChange: (e: React.ChangeEvent<any>) => void
+}
+
+const SimpleSearchRow: React.FC<SimpleSearchRowProps> = ({
+  field,
+  parameter,
+  index,
+  onChange
+}) => {
+  
+  const options = [
+    'Exact',
+    'Low Fuzziness',
+    'Medium Fuzziness',
+    'High Fuzziness'
+  ]
+
+  const [matchType, setMatchType] = useState<string>('')
+
+  function handleStrictLevelChange(event: React.ChangeEvent<any>) {
```

Comment:
Let us use the arrow function syntax :wink:

Reply:
Thanks. 
Just out of curiosity, is there a specific reason for that?

Reply:
It is just used for consistency, that's all :smile_cat: